examples/oft-solana/tasks/solana/sendOFT.ts

```diff
@@ -16,137 +16,121 @@ import {
+            )
+        }
 
-            const recipientAddressBytes32 = addressToBytes32(to)
+        const tokenAccountData = await fetchToken(umi, tokenAccount)
+        const balance = tokenAccountData.amount
 
-            const { nativeFee } = await oft.quote(
-                umi.rpc,
-                {
-                    payer: umiWalletSigner.publicKey,
-                    tokenMint: mint,
-                    tokenEscrow: umiEscrowPublicKey,
-                },
-                {
-                    payInLzToken: false,
-                    to: Buffer.from(recipientAddressBytes32),
-                    dstEid: toEid,
-                    amountLd: BigInt(amount),
-                    minAmountLd: 1n,
-                    options: Buffer.from(''),
-                    composeMsg: undefined,
-                },
-                {
-                    oft: oftProgramId,
-                }
+        if (amount == BigInt(0) || amount > balance) {
+            throw new Error(
+                `Attempting to send ${amount}, but ${umiWalletSigner.publicKey} only has balance of ${balance}`
             )
+        }
 
-            const ix = await oft.send(
-                umi.rpc,
-                {
-                    payer: umiWalletSigner,
-                    tokenMint: mint,
-                    tokenEscrow: umiEscrowPublicKey,
-                    tokenSource: tokenAccount[0],
-                },
-                {
-                    to: Buffer.from(recipientAddressBytes32),
-                    dstEid: toEid,
-                    amountLd: BigInt(amount),
-                    minAmountLd: (BigInt(amount) * BigInt(9)) / BigInt(10),
-                    options: Buffer.from(''),
-                    composeMsg: undefined,
-                    nativeFee,
-                },
-                {
-                    oft: oftProgramId,
-                    token: tokenProgramId,
-                }
-            )
+        const recipientAddressBytes32 = addressToBytes32(to)
 
-            let txBuilder = transactionBuilder().add([ix])
-            txBuilder = await addComputeUnitInstructions(
-                connection,
-                umi,
-                fromEid,
-                txBuilder,
-                umiWalletSigner,
-                computeUnitPriceScaleFactor,
-                TransactionType.SendOFT
-            )
-            const { signature } = await txBuilder.sendAndConfirm(umi)
-            const transactionSignatureBase58 = bs58.encode(signature)
+        const { nativeFee } = await oft.quote(
+            umi.rpc,
+            {
+                payer: umiWalletSigner.publicKey,
+                tokenMint: mint,
+                tokenEscrow: umiEscrowPublicKey,
+            },
+            {
+                payInLzToken: false,
+                to: Buffer.from(recipientAddressBytes32),
+                dstEid: toEid,
+                amountLd: BigInt(amount),
+                minAmountLd: 1n,
```

Comment:
Should we take this as a parameter instead and default it to amountLd?

Reply:
I can do this in a separate PR, since that line is not relevant to this PR

on the value, I did a search in monorepo and it seems that the standard is to set it to 90% of amountLD